### ISSUE TITLE
Connection Banner: Copy suggestions

### ISSUE THREAD

**Issue by oskosk** [Automattic/jetpack#11601]
In slack, @beaulebens observed and proposed a few things:

1. _The "Jetpack free protects" at the beginning for maybe change to "Jetpack protects... for free" as the former is a bit hard to parse_.
1. To maybe change `from our global network of WordPress.com servers` to just `from our global network of servers`.

cc @joanrho 

#### Currently
![image](https://user-images.githubusercontent.com/746152/54600813-f9f61980-4a13-11e9-9263-894166b8a3cf.png)

#### With the proposals

![image](https://user-images.githubusercontent.com/746152/54600944-45102c80-4a14-11e9-97eb-66052da90c8e.png)

#### Context

This is part of the [shortcopy project](https://github.com/orgs/Automattic/projects/82).

**Comment by keoshi** [comment]
Those changes make all the sense to me, particularly the first one!

**Comment by joanrho** [comment]
Agree! LGTM

**Comment by jeffgolenski** [comment]
Thanks @oskosk (and @beaulebens ). I'm corralling improvements for the connection banner and pages in this master issue. Our editorial team made some content improvements which also clean up these suggestions. Much appreciated!: https://github.com/Automattic/jetpack/issues/11614
